dart/lib/src/sentry_tracer.dart

```diff
@@ -284,6 +284,9 @@ class SentryTracer extends ISentrySpan {
 
   @override
   void setMeasurement(String name, num value, {SentryMeasurementUnit? unit}) {
+    if (finished) {
+      return;
```

Comment:
`l`: Maybe log a debug message.

Reply:
There's a separate issue to make the tracer loggable because we don't have the instance of the logger, but we'll do it soon.